mi/dataset/parser/auv_common.py

```diff
@@ -142,13 +158,13 @@ def parse_file(self):
                         self._exception_callback(RecoverableSampleException(msg))
                     else:
                         try:
-                            timestamp = compute_timestamp(parts)
+                            timestamp = _compute_timestamp(parts)
```

Comment:
`compute_timestamp(parts)`?

Reply:
The function in the self._auv_message_map  map variable (_compute_timestamp) should be called, not necessarily the compute_timestamp function at the top even though it's the only timestamp computation function defined.  Python's scoping probably would have taken care of this and adding the "_" in front of the variable compute_timestamp is probably not necessary, but I think it is clearer when reading it.